src/lib/Scenes/City/City.tsx

```diff
@@ -104,7 +102,6 @@ export class CityView extends Component<Props, State> {
   }
 
   setSelectedTab(selectedTab) {
-    this.setState({ selectedTab: selectedTab.i })
```

Comment:
This state doesn't actually get used by the component, but does trigger an extra render pass.

https://github.com/artsy/emission/pull/1406/files#r266063792